guides.md

```diff
@@ -69,6 +69,35 @@ members. Skip the steps below at your discretion for low-stakes topics.
   - as needed, send reminders of action items
   - migrate action items into task tracker
 
+## Managing shortlinks
+
+We use a custom shortlink service at `link.hypha.coop`. It helps us:
+- resolve keywords to URLs independantly of our personal workstation setup,
```

Comment:
I'm not sure I know that this bullet means @patcon --

Reply:
### 
```suggestion
- easily resolve keywords to URLs from any computer,
```